File: Auto-generated-tasks/Difficulty-2/Task-1/Solution.py
Kol = 3
i = 0
k = 0
Point = [0] * Kol;
for i in range(0, Kol):
	Point[i][i] = 1;
for i in range(0, Kol):
	for k in range(0, Kol):
		print(Point[i][k], end = " ")
	print("")

File: Auto-generated-tasks/Difficulty-2/Task-1/Solution.go
package "main"
import "fmt"
Kol:= 3
i:= 0
k:= 0
var Point [Kol][Kol]rune
for i = 0; i < Kol; i++ {
	for k = Kol; k < Kol; k++ {
		Point[i][k] = 0
	}
	Point[i][i] = 1
}
for i = 0; i < Kol; i++ {
	for k = Kol; k < Kol; k++ {
		fmt.Print(Point[i][k] + " ");
	}
	fmt.PrintLn("");
}